packages/networks/src/defaults/genesis.ts

```diff
@@ -53,8 +53,11 @@ export const knownGenesis: KnownGenesis = {
     '0xe3777fa922cafbff200cadeaea1a76bd7898ad5b89f7848999058b50e715f636', // Kusama CC2
     '0x3fd7b9eb6a00376e5be61f01abb429ffb0b104be05eaff4d458da48fcd425baf' // Kusama CC1
   ],
-  'nodle-chain': [
-    '0xa3d114c2b8d0627c1aa9b134eafcf7d05ca561fdc19fb388bb9457f81809fb23'
+  node: [ // This is Polkadex Mainnet
```

Comment:
```suggestion
  polkadex: [
```

Lint will probably complain about ordering after this